opendevin/runtime/plugins/agent_skills/agentskills.py

```diff
@@ -467,6 +467,12 @@ def _edit_file_impl(
 
     try:
         n_added_lines = None
+
+        # lint the original file
+        enable_auto_lint = os.getenv('ENABLE_AUTO_LINT', 'false').lower() == 'true'
+        if enable_auto_lint:
+            original_lint_error, _ = _lint_file(file_name)
```

Comment:
bug: this will change (and potentially) corrupt the original file. We should first copy it before linting.